Core/Utilities/Statistics.py

```diff
@@ -1,6 +1,5 @@
-##################################################################################################
```

Comment:
Delete if not used ? I guess we have numpy in DIRACOS that has this stuff and more

Reply:
I deleted it in v7r2